std/didexchange/v1/common.go

```diff
@@ -86,14 +86,18 @@ func (m *commonImpl) VerKey() (key string) {
 }
 
 func (m *commonImpl) Endpoint() (addr service.Addr) {
+	defer err2.Catch(err2.Err(func(err error) {
```

Comment:
**Just** FYI: with the current err2 allows this catch-block shortened, and `cmd/root.go` hast `glog.CopyStandardLogTo("ERROR")` which gives pretty much the same result:
```go
defer err2.Catch()
```
Why this is little better, it's the refactoring and copy-pasting, not the amount of lines some much.